content/sensu-go/5.0/getting-started/sample-app.md

```diff
@@ -0,0 +1,363 @@
+After saving, you'll see your webhook URL under Integration Settings.
+
+Open `go/config/handlers/slack.yaml` and replace `SECRET` in the following line with your Slack workspace webhook URL:
+
+{{< highlight shell >}}
+"command": "slack-handler --channel '#demo' --timeout 20 --username 'sensu' --webhook-url 'SECRET'",
+{{< /highlight >}}
+
+So it looks something like:
+
+{{< highlight shell >}}
+"command": "slack-handler --channel '#demo' --timeout 20 --username 'sensu' --webhook-url 'https://hooks.slack.com/services/XXXXXXXXXXXXXXXX'",
+{{< /highlight >}}
+
+**3. Create a handler to send alert events to Slack that uses the `slack-handler` asset.**
+
+{{< highlight shell >}}
+sensuctl create -f go/config/handlers/slack.yaml
+{{< /highlight >}}
+
+### Create a Sensu service check to monitor the status of the dummy app
+
+To automatically monitor the status of the dummy app, we'll create an asset that lets the Sensu agents use a [Sensu HTTP plugin](https://github.com/portertech/sensu-plugins-go).
+
+**1. Create the `check-plugins` asset.**
+
+{{< highlight shell >}}
+sensuctl create -f go/config/assets/check-plugins.yaml
+{{< /highlight >}}
+
+**2. Now we can create a check to monitor the status of the dummy app that uses the `check-plugins` asset and the Slack pipeline.**
+
+{{< highlight shell >}}
+sensuctl create -f go/config/checks/dummy-app-healthz.yaml
+{{< /highlight >}}
+
+**3. With the automated alert workflow in place, we can see the resulting events in the Sensu dashboard.**
+
+At http://webui.sensu.local/signin, sign in with your sensuctl username (`demo`) and password.
+
+**4. Toggle the health of the dummy app to simulate a failure.**
+
+{{< highlight shell >}}
+curl -iXPOST http://dummy.local/healthz
+{{< /highlight >}}
+
+We should now be able to see a critical alert in the [Sensu dashboard](http://webui.sensu.local/events) as well as by using sensuctl:
+
+{{< highlight shell >}}
+sensuctl event list
+{{< /highlight >}}
+
+You should also see an alert in Slack.
+
+Continue to post to `/healthz` until all Sensu entities return to a healthy state.
+
+{{< highlight shell >}}
+curl -iXPOST http://dummy.local/healthz
+{{< /highlight >}}
+
+## Collecting app metrics
+
+### Create a Sensu service check to collect Prometheus metrics
+
+To automatically collect the Prometheus metrics from the dummy app, we'll create an asset that lets the Sensu agents use the [Sensu Prometheus plugin](https://github.com/sensu/sensu-prometheus-collector).
+
+**1. Create the `prometheus-collector` asset.**
+
+{{< highlight shell >}}
+sensuctl create -f go/config/assets/prometheus-collector.yaml
+{{< /highlight >}}
+
+**2. Now we can create a check to collect Prometheus metrics that uses the `prometheus-collector` asset.**
+
+{{< highlight shell >}}
+sensuctl create -f go/config/checks/dummy-app-prometheus.yaml
+{{< /highlight >}}
+
+### Use Grafana to visualize metrics
```

Comment:
Ah, ok. So this is where the sidecar comes in and why I didn't see it in the earlier `entity list`